submissions/annStoyano/html-css-popup/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navigation">
+            <ul class="nav-items">
+                <li><a href="#" class="mails link">Mails</a></li>
+                <li><a href="#" class="mails link">Images</a></li>
+                <li class="popup-wrapper">
+                    <input type="checkbox" id='popup-check' class='popup_btn'>
+                    <label class="popup_btn-icon" for='popup-check'></label>
+                    <div class="popup">
+                        <div class="popup-content">
+                            <section class="main-menu">
+                                <a href="#" class="popup-item">
```

Comment:
Maybe someone of mentors will advise you here to use this tag (`section`) correctly
I would use the usual `div`